Documentation/security/network/index.rst

```diff
@@ -5,10 +5,9 @@
     https://docs.cilium.io
 
 .. _concepts_security:
-.. _security_root:
```

Comment:
This is in the 3rd commit, but we want to move it to the 1st commit, or we'll have commits 1 and 2 with duplicate reference targets for `_security_root`.